pywbem/_logging.py

```diff
@@ -117,279 +167,183 @@
 DEFAULT_LOG_DESTINATION = 'none'
 
 
-class MetaPywbemLoggers(type):
-    """
-    **Experimental:** *New in pywbem 0.11 as experimental.*
-
-    This metaclass allows the definition of __str__ and __repr__ at the
-    class level in the subclass.
-    """
-    loggers = {}
-
-    def __str__(cls):
-        return 'PywbemLoggers(loggers={s.loggers!r})'.format(s=cls)
-
-    def __repr__(cls):
-        return 'PywbemLoggers({s.loggers!r})'.format(s=cls)
-
-
-# pylint: disable=no-init
-class PywbemLoggers(six.with_metaclass(MetaPywbemLoggers)):
+def define_loggers_from_string(log_spec_str, log_filename=None):
+    # pylint: disable=line-too-long
     """
-    **Experimental:** *New in pywbem 0.11 as experimental.*
+    Create the pywbem loggers defined by the input string in the following
+    syntax and place the logger definitions in the class level dictionary
+    in this class.
```

Comment:
"This class" and "class level dictionary" are terms that are still from the PywbemLoggers class.

Reply:
Done

Reply:
they are now gone